prettier.config.mjs

```diff
@@ -2,14 +2,4 @@ import config from '@ibexa/eslint-config/prettier';
 
 export default {
     ...config,
-    overrides: [
```

Comment:
At first I've added this parser to test how it works and if it helps, but it has some questionable ways of formatting code that just doesn't look right, therefore I think it failed the test.